src/bin/pg_autoctl/keeper.c

```diff
@@ -1200,8 +1224,7 @@ keeper_register_and_init(Keeper *keeper, NodeState initialState)
 		 * prevent trying to init again and cause strange errors.
 		 */
 		unlink_file(config->pathnames.state);
```

Comment:
This call is now unnecessary. Let's move the comment to the `rollback` section.

Reply:
Good point, done that way now.